external_executors/README.md

```diff
@@ -0,0 +1,3 @@
+# External Executors
+
+Docker images for running workflows in external environments.
```

Comment:
Some ideas for this part are:
- Point to the portability documents
--
 https://docs.google.com/document/d/1yF9Vlhr1gbFOS3nmVa7RH5umBPeuDtsdzTdaluIJNEc/edit#heading=h.9w3s9v6r318z
-- https://docs.google.com/document/d/1ghLoHMbKOPsndA1WgdSAHm5X82p86ryLBiAt1hz6HuI/edit

- Quick description of implement environments.
- A section placeholder for instructions to add new execution services.
- A section placeholder on how to submit to the service.
- Any link Brian thinks is helpful to point to the GA4GH portability work and an indicator this is a driver project.